packages/vscode-extension/lib/inspector.js

```diff
@@ -0,0 +1,118 @@
+const { Dimensions, findNodeHandle } = require("react-native");
+
+function requireGetInspectorDataForViewAtPoint() {
+  return require("react-native/Libraries/Inspector/getInspectorDataForViewAtPoint");
```

Comment:
We should just put require call where this method is used. Very little benefit from extracting it to a method

Reply:
moved to the method